src/components/Match/MatchStory.jsx

```diff
@@ -540,6 +557,18 @@ const generateStory = (match) => {
   // Intro
   events.push(new IntroEvent(match));
 
+  // Prediction
+  let predExists = false;
+  match.players.forEach((player) => {
+    if (player.pred_vict === true) {
+      predExists = true;
+    }
+  });
+  if (predExists === true) {
+    events.push(new PredictionEvent(match, -89));
+    events.push(new PredictionEvent(match, -88));
```

Comment:
that is intentional

![onepredictionexample](https://cloud.githubusercontent.com/assets/8967168/25873141/4811922a-34db-11e7-85a7-b043231182c4.png)